nixos/modules/services/web-servers/minio.nix

```diff
@@ -69,6 +70,17 @@ in
       type = types.package;
       description = "Minio package to use.";
     };
+
+    extraConfig = mkOption {
+        type = types.attrsOf types.str;
+        default = {};
+        example = { MINIO_ROOT_USER = "minio"; };
```

Comment:
Not the best example seeing as it would be entirely redundant to add this here... Do we actually need this `extraConfig` option? If it isn't common to need I would suggest letting the sysadmin directly reference `systemd.services.minio.environment` instead. You could even mention in the documentation :man_shrugging:

Reply:
`systemd.services.minio.environment` might be an option. I'm not sure what's more idiomatic here?

Reply:
Some people are a fan of adding these options. I think it adds value if its going to be very common, but otherwise it just adds noise. But my opinion isn't all that important, so you should go with what you think. I just thought I'd raise the point for discussion.

Reply:
I remove this here can be done in a sparate MR